javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,69 @@
+let yearOfBirth = 1995;
+let yearFuture = 2027;
+let age = yearOfBirth - yearFuture;
+console.log("You will be " + age + "years old in " + yearFuture + ".");
+
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2021;
+let dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears == true) {
+  console.log("Your dog will be " + dogYear + "years old in" + dogYearFuture);
+} else {
+  console.log("Your dog will be " + age + "years old in" + dogYearFuture);
+}
+
+let width;
+let depth;
+let height;
+let volumeInMeters = width * depth * height;
+let gardenSizeInM2;
+let fairHousePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+let actualHousePrice;
+
+if (actualHousePrice >= fairHousePrice) {
+  console.log("The prospective buyers are paying too much for the house.");
+} else {
+  console.log("The prospective buyers are paying a fair amount for the house.");
+}
```

Comment:
Just for the sake of easier troubleshooting, write the numbers you are getting into the output, f.ex 
"The prospective buyers are paying {500000} for their house. This is  a fair amount." or 
"The prospective buyers are paying {500000} for the house. This is too much, it should only cost {300000}"

I know this is outside the scope of the exercise, but it could help you see when you have errors.